### PR TITLE
update the ddf inplace when remove columns for mutable ddf

### PR DESCRIPTION
**Pull request by PangZhi**
This PR is to solve the problem that Remove Column in ddf should do in place replacement for mutable ddf.

/cc @hai-adatao @nhanitvn @Huandao0812

### CODE REVIEW

**Comment by Huandao0812** [comment]
updated, @PangZhi pls fix the UT

File: jdbc/src/main/scala/io/ddf/jdbc/content/ViewHandler.scala [hunk 1]
```diff
@@ -32,8 +32,8 @@ class ViewHandler(ddf: DDF) extends io.ddf.content.ViewHandler(ddf) {
     }
     }
     val newddf: DDF = this.project(result)
```
Comment (hai-adatao) [inline jdbc/src/main/scala/io/ddf/jdbc/content/ViewHandler.scala]: this line is redundant

**Comment by hai-adatao** [comment]
@PangZhi I rerun the tests thrice and it still fail with the same error, can you help investigate?

**Comment by hai-adatao** [comment]
UT passed now, thanks @PangZhi

**Pull request merged**